site/content/en/docs/Installation/upgrading.md

```diff
@@ -51,6 +51,74 @@ The following are steps to implement this:
 If you are upgrading a single cluster, we recommend creating a maintenance window, in which your game goes offline
 for the period of your upgrade, as there will be a short period in which Agones will be non-responsive during the upgrade.
 
+{{% feature publishVersion="1.43.0" %}}
+#### In-Place Agones Upgrades
+
+{{< alert color="warning" title="Warning" >}}
+Work is ongoing for [In-Place Agones Upgrades](https://github.com/googleforgames/agones/issues/3766),
+and the feature is currently in `Dev`. Please continue to use the multi-cluster strategy for
+production critical upgrades. Feedback on the upcoming `Alpha` and `Beta` releases is appreciated.
+{{< /alert >}}
+
+For In-Place Agones Upgrades we highly recommend installing using Helm. Helm has a significant
+advantage over `install.yaml` in that Helm automatically rolls back the upgrade if the agones-system
+pods are not all successfully running within the designated `--timeout`.
+
+Note: By “controller derived configuration” we mean the parts of the Game Server’s final state that
+are not part of the Game Server spec template that are instead passed to the Game Server through the
+controller, such as the FEATURE_GATES or the Agones SDK Image.
+
+##### Upgrade Existing Fleet (recommended)
+
+1. Run `helm upgrade my-release --install --atomic --wait --timeout 10m --namespace=agones-system`
+with all the appropriate arguments, such a `--version`, for your specific upgrade. Keep in mind that
+`helm upgrade` overwrites all `--set agones.` arguments, so these must be set for each upgrade. See
+[Helm Configuration]({{< relref "./Install Agones/helm.md" >}}) for a list of all available
+configurable parameters.
+2. Wait until the `helm upgrade` is complete.
+3. Kick off rolling update of existing Fleet.
+    1. Make any change to the Fleet Spec Template in your fleet.yaml and reapply with
+    `kubectl apply -f fleet.yaml`.
+    2. If you have no changes to make to the existing Fleet Spec Template in the fleet.yaml,
+    then add a label to the spec.template.metadata.labels. This has the added advantage that Game
```

Comment:
Either one will kick off a Fleet update. I'm suggesting the labels here, so that they can use `matchLabels` to create a gameserverallocation like:

```
apiVersion: allocation.agones.dev/v1
kind: GameServerAllocation
spec:
  selectors:
    - matchLabels:
        release: 1.42.0
```

Reply:
Nit on rewrite:

```markdown
If you have no changes to make to the existing `spec.template` in the fleet.yaml, then either add a label or annotation to the `spec.template.metadata`. Reapply with `kubectl apply -f fleet.yaml`.
```

(also being consistent on how we're referencing each part of the CRD)

I don't know if we need to be providing recommendation on label vs annotation here -- just because it's entirely up to the end user -- they may not care about filtering on it on allocation and it is a simpler instruction without it.

WDYT?